lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -526,11 +549,16 @@ def __init__(self, verts, *args, **kwargs):
     }
 
     def set_zsort(self, zsort):
-        '''
-        Set z-sorting behaviour:
-            boolean: if True use default 'average'
-            string: 'average', 'min' or 'max'
-        '''
+        """
+        Sets the calculation method for the z-order.
+
+        Parameters
+        ----------
+        zsort : bool or {'average', 'min', 'max'}
+            For 'average', 'min', 'max' the z-order is determined by applying
+            the function to the z-orders of the individual polygons. *True* is
```

Comment:
"the function to the z coordinates of the vertices in the viewer's coordinate system."

Perhaps clearer, more precise. Z-order as a term is a bit loaded in matplotlib.